### PR TITLE
Added page for median absolute deviation

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Adds a page for Median Absolute Deviation documentation

image
image

### Issues Resolved
(Won't link for now)

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _aggregations/metric/median-absolute-deviation.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    "hits": []
+  },
+  "aggregations": {
+    "median_absolute_deviation_distanceMiles": {
+      "value": 1829.8993624441966
+    }
+  }
+}}
+
+
```
Comment (Yury-Fridlyand) [inline _aggregations/metric/median-absolute-deviation.md]: A typo?
```suggestion
}
```

**Review by Yury-Fridlyand — APPROVED** [review]
Wow!
Can you render this page and include to the PR as pdf?

GHA finds couple of errors - please have a look on them too

File: _aggregations/metric/median-absolute-deviation.md (hunk 1 above)
Comment (MitchellGale) [reply _aggregations/metric/median-absolute-deviation.md]: Thanks! Missed that :)

File: _aggregations/metric/median-absolute-deviation.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+# Median Absolute Deviation Aggregations
+
+The `median_absolute_deviation` metric is a single-value metric aggregation that returns median absolute deviation field. Median absolute deviation is a statistical measure of data variability. It is used to measure dispersion from the median, may be less impacted by outliers in a dataset. 
+
+Median absolute deviation is calculated with:<br>
+median_absolute_deviation = median(|X<sub>i</sub> - Median(X<sub>i</sub>)|)
+
+The following example calculates the median absolute deviation of the `DistanceMiles` field of the opensearch_dashboards_sample_data_flights index:
```
Comment (GumpacG) [inline _aggregations/metric/median-absolute-deviation.md]: Should be `opensearch_dashboards_sample_data_flights`

File: _aggregations/metric/median-absolute-deviation.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  "aggregations": {
+    "median_absolute_deviation_distanceMiles": {
+      "value": 1829.8993624441966
+    }
+  }
+}
+```
+
+### Missing
+If there are documents in your index that are missing fields and you wish to set a default value for those documents, use the `missing` parameter to specify the default value for any missing fields from documents.
```
Comment (GumpacG) [inline _aggregations/metric/median-absolute-deviation.md]: To use the active voice as specified in https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#voice, we can modify to have:
"You can set a default value for missing fields from documents by specifying the `missing` parameter.

Does this also replace `NULL` values?

File: _aggregations/metric/median-absolute-deviation.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 65
+redirect_from:
+  - /query-dsl/aggregations/metric/median-absolute-deviation/
+---
+
+# Median Absolute Deviation Aggregations
+
+The `median_absolute_deviation` metric is a single-value metric aggregation that returns median absolute deviation field. Median absolute deviation is a statistical measure of data variability. It is used to measure dispersion from the median, may be less impacted by outliers in a dataset.
```
Comment (GumpacG) [inline _aggregations/metric/median-absolute-deviation.md:13]:
```suggestion
The `median_absolute_deviation` metric is a single-value metric aggregation that returns median absolute deviation of a field. Median absolute deviation is a statistical measure of data variability. It is used to measure dispersion from the median and may be less impacted by outliers in a dataset. 
```

File: _aggregations/metric/median-absolute-deviation.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+layout: default
+title: Median Absolute Deviation
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 65
+redirect_from:
+  - /query-dsl/aggregations/metric/median-absolute-deviation/
+---
+
+# Median Absolute Deviation Aggregations
```
Comment (GumpacG) [inline _aggregations/metric/median-absolute-deviation.md:11]:
```suggestion
# Median absolute deviation aggregations
```

File: _aggregations/metric/median-absolute-deviation.md (hunk 3 above)
Comment (GumpacG) [reply _aggregations/metric/median-absolute-deviation.md]: For context, it also replaces NULL in bucket aggregations https://opensearch.org/docs/latest/aggregations/bucket/missing/

File: _aggregations/metric/median-absolute-deviation.md (hunk 2 above)
Comment (MitchellGale) [inline _aggregations/metric/median-absolute-deviation.md]:
```suggestion
The following example calculates the median absolute deviation of the `DistanceMiles` field of the `opensearch_dashboards_sample_data_flights` index:
```

Comment (MitchellGale) [reply _aggregations/metric/median-absolute-deviation.md]: Thanks, updated.

**Review by GumpacG — APPROVED** [review]

File: _aggregations/metric/median-absolute-deviation.md (hunk 3 above)
Comment (MitchellGale) [reply _aggregations/metric/median-absolute-deviation.md]: Added details about null and added more active voice

**Pull request merged**